src/components/diff/Diff.tsx

```diff
@@ -14,17 +15,30 @@ const DIFF_PROVIDER_REGISTRY: { [key: string]: typeof React.Component } = {
 /**
  * Determines if a given file is supported for diffs.
  *
- * This will be removed when "Plaintext" diffs are supported since that will be used
- * for cases where there is not a dedicated diff provider.
- *
  * @param path the file path
  */
 export function isDiffSupported(path: string): boolean {
-  return PathExt.extname(path).toLocaleLowerCase() in DIFF_PROVIDER_REGISTRY;
+  return (
+    PathExt.extname(path).toLocaleLowerCase() in DIFF_PROVIDER_REGISTRY ||
+    isText(path)
```

Comment:
Even with `isText` added this is still not a great way to determine diff-ability. The problem is that it still misses many plaintext files named with weird or missing extensions that do handled correctly by eg `git diff`.

We can save this problem for a later PR, but I wanted to point it out here

Reply:
Thanks for the creation of the related issue